src/lib.rs

```diff
@@ -38,6 +39,30 @@ use embedded_text::{
 
 pub use embedded_menu_macros::{Menu, SelectValue};
 
+/// Internal function to change the selection based on interaction. Moved
+/// separated to allow for easier testing.
+fn calculate_selection(selected: usize, count: usize, interaction: InteractionType) -> usize {
```

Comment:
Hmm, I think this could maybe be a `pub(crate)` method on InteractionType, but never mind that :)

Reply:
I can still do that. I have one more change before I'd like a release.

Reply:
I'll leave it up to you, I'm fine with it as it is :)